doc/source/ray-overview/index.md

```diff
@@ -34,273 +34,13 @@ To use Ray in Java, first add the [ray-api](https://mvnrepository.com/artifact/i
 Want to build Ray from source or with docker? Need more details? 
 Check out our detailed [installation guide](installation.rst).
 
+## Ray AI Runtime Quick Start
 
-## Ray ML Quick Start
+Ray AI Runtime (AIR) is an open-source toolkit for building end-to-end ML applications.
```

Comment:
@richardliaw with the ML libs factored out, I wonder how we can make sure this "top level" getting started guide is useful as a first touch point. It would also be ok if we were to slightly repeat ourselves in this guide and e.g. the AIR one. Better to have some redundancy, than to have users search for info.